source/crud/tests/unified/aggregate.json

```diff
@@ -42,10 +42,6 @@
         {
           "_id": 3,
           "x": 33
-        },
```

Comment:
The cursor actually needs to do another getmore after the final document in the result set, so we only need two documents to generate the expected getmore events.

Reply:
For the record, this test was only introduced in https://github.com/mongodb/specifications/commit/3586db8b1ed90d835737f6226bb3ff9012cfc5de#diff-665043dea9fd9951f2063ea5827d4e888cfdfe44e7deda6201f688e63f9dadc0 for #964 so most drivers didn't catch this yet?

I just want to make sure we don't need to look into any drivers that were incorrectly passing the previous test by inadvertently ignoring extra observed events.

Reply:
I don't think any drivers have pulled in that test yet since it's also part of DRIVERS-1603, which does not yet have unblocked language tickets.